test/e2e/testing-manifests/kubectl/nginx-statefulset.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: v1
+kind: Service
```

Comment:
noted! but will statefulset work correctly in e2e environment if it does not have a headless service?